scripts/backend_tests_incomplete_coverage.txt

```diff
@@ -77,6 +77,8 @@ core.jobs.job_test_utils_test
 core.jobs.job_utils_test
 core.domain.image_validation_services_test
 scripts.linters.js_ts_linter_test
+core.storage.improvements.gae_models_test
+core.storage.audit.gae_models_test
```

Comment:
Why are you adding these two files?

Reply:
I think that mistakenly occurred when I pulled the latest changes. Will correct them.